server/src/collection.rs

```diff
@@ -95,4 +95,16 @@ impl Collection {
     pub fn path(&self) -> &OwnedObjectPath {
         &self.path
     }
+
+    pub async fn label(&self) -> String {
+        self.label.lock().await.clone()
+    }
```

Comment:
This would be gone after your rebase, as it is a property ;)

Reply:
Oh! I had already implemented these changes (signals and properties) locally. Since you said smaller prs. I just didn't wanted to trouble you with many prs at once :)

Reply:
Rebased.